README.md

```diff
@@ -67,9 +72,30 @@ _Note: binaries available via `cargo install` is coming soon._
 
 ## Getting started
 
-We recommend to browse the [examples](https://github.com/rs-ipfs/rust-ipfs/tree/master/examples) and
-[tests](https://github.com/rs-ipfs/rust-ipfs/tree/master/tests) in order to see how to use Rust-IPFS
-in different scenarios.
+We recommend browsing the [examples](https://github.com/rs-ipfs/rust-ipfs/tree/master/examples), the [http crate tutorial](https://github.com/rs-ipfs/rust-ipfs/tree/master/http#getting-started) and [tests](https://github.com/rs-ipfs/rust-ipfs/tree/master/tests) in order to see how to use Rust-IPFS in different scenarios.
+
+### Running the tests
+
+The project currently features unit, integration, conformance and interoperability tests. Unit and integation tests can be run with:
+
+```bash
+$ cargo test
+```
+
+Explanations on how to run the conformance tests can be found [here](https://github.com/rs-ipfs/rust-ipfs/tree/master/conformance). The Go and JS interoperability tests are behind a feature flag and can be run with:
+
+```bash
+$ cargo test --feature=test_go_interop
+$ cargo test --feature=test_js_interop
+```
+
+Note: you will need to set the `GO_IPFS_PATH` and the `JS_IPFS_PATH` environment variables to point to the relevant IPFS binary.
```

Comment:
There could be another note that these features cannot currently be enabled at the same time (`cargo test --all-features` for example).